README.md

```diff
@@ -731,11 +731,15 @@ This is useful if you do not want to explicitly set up your registry
 configuration in all of your package.json files individually when e.g. using
 private registries.
 
-#### --skip-temp-tag
+#### --skip-temp-tag (DEPRECATED)
```

Comment:
I wanted to use `--temp-tag` because yargs has a [negate-fields](https://github.com/yargs/yargs#negate-fields) feature where one could say `--no-temp-tag` and that would work better than `--no-use-temp-tag`.

If this isn't importand I can switch to the one you requested.